### ISSUE TITLE
'async' is a reserved word in Python >= 3.7

### ISSUE THREAD

**Issue by cclauss** [openai/baselines#495]
flake8 testing of https://github.com/openai/baselines on Python 3.7.0

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./baselines/bench/monitor.py:142:55: F821 undefined name 'uuid'
    mon_file = "/tmp/baselines-test-%s.monitor.csv" % uuid.uuid4()
                                                      ^
./baselines/bench/monitor.py:158:20: F821 undefined name 'pandas'
    last_logline = pandas.read_csv(f, index_col=None)
                   ^
./baselines/bench/monitor.py:161:5: F821 undefined name 'os'
    os.remove(mon_file)    ^
./baselines/acktr/acktr_cont.py:57:69: E999 SyntaxError: invalid syntax
                                epsilon=1e-2, stats_decay=0.99, async=1, cold_iter=1,
                                                                    ^
./baselines/acktr/value_functions.py:24:41: E999 SyntaxError: invalid syntax
                                    async=1, kfac_update=2, cold_iter=50, \
                                        ^
./baselines/acktr/acktr_disc.py:62:39: E999 SyntaxError: invalid syntax
                stats_decay=0.99, async=1, cold_iter=10, max_grad_norm=max_grad_norm)
                                      ^
./baselines/acktr/kfac.py:13:164: E999 SyntaxError: invalid syntax
    def __init__(self, learning_rate=0.01, momentum=0.9, clip_kl=0.01, kfac_update=2, stats_accum_iter=60, full_stats_init=False, cold_iter=100, cold_lr=None, async=False, async_stats=False, epsilon=1e-2, stats_decay=0.95, blockdiag_bias=False, channel_fac=False, factored_damping=False, approxT2=False, use_float64=False, weight_decay_dict={},max_grad_norm=0.5):
                                                                                                                                                                   ^
4     E999 SyntaxError: invalid syntax
3     F821 undefined name 'uuid'
7
```

**Comment by wookayin** [comment]
How about `is_async`? I think it would be a better name than `async_` (which is somewhat artificial).

**Comment by cclauss** [comment]
I see other repos replacing __async__ with __non_blocking__.

**Comment by cclauss** [comment]
Fixed in #542
